### PR TITLE
Jb41724 sort uploads

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

**Comment by blammit** [comment]
@saukko

File: src/sailfishui/plugin/pendinguploadsmodel.cpp [hunk 1]
```diff
@@ -129,7 +129,13 @@ void PendingUploadsModel::setData(const QStringList &data)
...
         item.dateCreated = QFileInfo(filePath).created();
 
-        d->contents.append(item);
+        int i = 0;
+        for (; i < d->contents.count(); ++i) {
+            if (item.dateCreated > d->contents.at(i).dateCreated) {
+                break;
+            }
+        }
+        d->contents.insert(i, item);
```
Comment (pvuorela) [inline src/sailfishui/plugin/pendinguploadsmodel.cpp:138]: Now the beginInsertRows values would be wrong if the insertions don't happen at the end?

Comment (blammit) [reply src/sailfishui/plugin/pendinguploadsmodel.cpp:137]: Oops, fixed now.

**Comment by pvuorela** [comment]
LGTM

**Comment by blammit** [comment]
Thanks @pvuorela are you able to merge as well?

**Comment by pvuorela** [comment]
Sure!

**Pull request merged**